src/EditorFeatures/CSharpTest/CodeActions/MoveType/MoveTypeTests.RenameFile.cs

```diff
@@ -34,6 +34,28 @@ class Inner { }
         }
 
         [WpfFact, Trait(Traits.Feature, Traits.Features.CodeActionsMoveType)]
+        public async Task TestRenameFileWithFolders()
+        {
+            var code =
+@"
+<Workspace>
+    <Project Language=""C#"" AssemblyName=""Assembly1"" CommonReferences=""true"">
+        <Document Folders=""A\B""> 
```

Comment:
nit: if Document tag supports giving it a name, specifying the name here makes test clearer (file name <> type name)